src/components/common/machinesConnectionSelector.tsx

```diff
@@ -99,16 +101,21 @@ export const MachinesConnectionSelector = ({
                        }
                        : {}
                    }>
-            <Radio isChecked={connectionName === LIBVIRT_SYSTEM_CONNECTION}
-                   onChange={() => onValueChanged('connectionName', LIBVIRT_SYSTEM_CONNECTION)}
-                   name="connectionName"
-                   id="connectionName-system"
-                   label={rephraseUI("connections", LIBVIRT_SYSTEM_CONNECTION)} />
-            <Radio isChecked={connectionName == LIBVIRT_SESSION_CONNECTION}
-                   onChange={() => onValueChanged('connectionName', LIBVIRT_SESSION_CONNECTION)}
-                   name="connectionName"
-                   id="connectionName-session"
-                   label={rephraseUI("connections", LIBVIRT_SESSION_CONNECTION)} />
+            { isDisabled
+                ? rephraseUI("connections", connectionName)
+                : <>
+                    <Radio isChecked={connectionName === LIBVIRT_SYSTEM_CONNECTION}
+                        onChange={() => onValueChanged('connectionName', LIBVIRT_SYSTEM_CONNECTION)}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-1917-76ac05de-20250915-164354-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)